src/core/server/elasticsearch/cluster_client.ts

```diff
@@ -151,8 +170,9 @@ export class ClusterClient {
    * scoped client instances, these will be automatically closed as soon as the
    * original cluster client isn't needed anymore and closed.
    * @param req - Request the `ScopedClusterClient` instance will be scoped to.
+   * Supports request optionality, Legacy.Request & FakeRequest for BWC with LegacyPlatform
    */
-  public asScoped(req: { headers?: Headers } = {}) {
+  public asScoped(request?: KibanaRequest | Request | FakeRequest) {
```

Comment:
`request?` for BWC with the current code. what is the case when cluster.asScoped(undefined) is valid ?